diagnostic.md

```diff
@@ -150,7 +168,13 @@ div {
 ```
 
 <!-- your answer starts here -->
+divOne will float to the right, divFour will float to the left, and divTwo, divThree and divFive will be stacked in the center.
+
+layout would look like this:
 
+divFour      divTwo       divOne
+            divThree
+            divFive
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```